### ISSUE TITLE
add sshd KexAlgorithms

### ISSUE THREAD

**Issue by simonkowallik** [F5Networks/f5-declarative-onboarding#208]
### Is your feature request related to a problem? Please describe.
This request is about including `KexAlgorithms` sshd_config option in DO.

### Describe the solution you'd like
Include `KexAlgorithms` just like the existing `MACS` and `ciphers`. KexAlgorithms is also relevant for security (and therefore compliance)

### Describe alternatives you've considered
Configure `KexAlgorithms` outside the scope of DO, but this makes things more complicated, especially because this might conflict with the DO settings.

**Comment by amolari** [comment]
Support Request 00277653 opened. We would need this implemented for compliance (CIS F5 Benchmarks).
Probably as easy as extending the Schema and add this:
```
    if (sshd.kex) {
        includeString = includeString.concat(`Ciphers ${sshd.kex.join(',')}\n`);
    }
```
to
https://github.com/F5Networks/f5-declarative-onboarding/blob/master/src/lib/systemHandler.js#L1134

**Comment by sunitharonan** [comment]
Thank you. I have added this issue to our internal product backlog as AUTOTOOL-3506

**Comment by amolari** [comment]
F5 Support case: 00277653 => RFE ID1167737

**Comment by sunitharonan** [comment]
This has been resolved in the 3.41.0 release of AS3.
